example/e2e/sanityTest.spec.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ */
+/* eslint-env jest */
+/* global device, element, by */
+
+describe('NetInfo', () => {
+  beforeEach(async () => {
+    await device.reloadReactNative();
+  });
+
+  it('should load example app with no errors and show all the examples', async () => {
+    await expect(element(by.id('examples'))).toExist();
+    await expect(element(by.id('example-isConnected'))).toExist();
+    await expect(element(by.id('example-currentInfoSingle'))).toExist();
+    await expect(element(by.id('example-currentInfoHistory'))).toExist();
+    await expect(element(by.id('example-isConnectionExpensive'))).toExist();
```

Comment:
Do these example elements output some text showing the NetInfo values? If so, it could be useful to use `.toHaveText()` to confirm the text that's shown. That ensures that real correct values are returned in the context of a running RN app.

Integrating with external resources can be risky in the context of a test, but for something as straightforward as NetInfo it may run reliably—I would assume that it would always report that the network connection is connected. If not, then there probably isn't a good way to test NetInfo in the context of an end-to-end test.

Reply:
@CodingItWrong Thanks for the comment! The idea of this PR is to get the foundations of e2e testing in with the bare minimum and then add the actual tests. The approach you mentioned is how I expect to get it working 👍 We also *might* be able to do actual testing of network conditions on the Android Emulator to get full e2e tests on that platform. iOS will be impossible because we can't alter the network state of the Simulator 👎